src/composeWithMongooseDiscriminators.js

```diff
@@ -0,0 +1,379 @@
+/* @flow */
+
+import type { ComposeFieldConfigMap } from 'graphql-compose';
+import {
+  EnumTypeComposer,
+  graphql,
+  schemaComposer,
+  SchemaComposer,
+  TypeComposer,
+} from 'graphql-compose';
+import type {
+  ComposePartialFieldConfigAsObject,
+  RelationOpts,
+} from 'graphql-compose/lib/TypeComposer';
+import type { GraphQLFieldConfigMap } from 'graphql-compose/lib/graphql';
+import { Model } from 'mongoose';
+import type { TypeConverterOpts } from './composeWithMongoose';
+import { composeWithMongoose } from './composeWithMongoose';
+import { recomposeBaseResolvers } from './utils/recomposeBaseResolvers';
+import { recomposeChildResolvers } from './utils/recomposeChildResolvers';
+
+const { GraphQLInterfaceType } = graphql;
+
+export type Options = {
+  reorderFields: string[] | boolean, // true order: _id, DKey, DInterfaceFields, DiscriminatorFields
+  customizationOptions: TypeConverterOpts,
+};
+
+// Enum MongooseComposeResolvers
+export const EMCResolvers = {
```

Comment:
We should avoid using `export const EMCResolvers = {` here.
If you want to create such mapping, you need to do it in https://github.com/graphql-compose/graphql-compose-mongoose/blob/master/src/resolvers/index.js

I'll refactor `connection` and `pagination` resolvers in upcoming days, so all resolvers will be under `/src/resolvers` folder.

Reply:
@mernxl Moved `connection` and `pagination` resolvers to the `/src/resolvers` folder.
`yarn install` is required! Was updated `graphql-compose-pagination` and `graphql-compose-connection` packages.